### ISSUE TITLE
Please consider changing `whitelist` and `blacklist` to something else

### ISSUE THREAD

**Issue by matthewhelmke** [greatsuspender/thegreatsuspender#1190]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: all
 * Browser name & version: all
 * Operating system & version: all

And please also do a search for your request/bug before create a new one thanks!
* I did not find a similar issue posted

My request is to use something like `allow list` and `deny list`.

**Comment by ossilator** [comment]
i'll note the idea that the underlying metaphor of these terms is somehow racist is ahistoric garbage. but whatever floats your boat, and maybe the (mostly inactive) maintainer feels like granting your wish.

also, it would have been wise to add (a link to) a rationale in the first place - someone who didn't catch the recent renaming hubbub would have given you a blank stare at the request.

**Comment by fstutzman** [comment]
I would disagree with the notion that those terms do not have racial connotations.  See e.g. https://tools.ietf.org/id/draft-knodel-terminology-00.html#rfc.section.1.2

For reference, please see that Apple, Microsoft, and Github amongst others are no longer using this terminology, because of the inherent racialization of the language.

https://www.cnet.com/news/apple-banishes-blacklist-and-master-branch-in-push-for-inclusive-language/

I would vote for "Allow List" and be comfortable that the user can figure it out.

**Comment by fstutzman** [comment]
I have created the pull request.  We will see if it is merged!

https://github.com/greatsuspender/thegreatsuspender/pull/1195

**Comment by ossilator** [comment]
i'm sorry, but that section in the rfc reads in parts like satire. clearly, the authors have no honkin' clue about the etymology of the words they're talking about.
the cnet article is basically just an appeal to authority. that a bunch of companies thought it prudent to follow the demand doesn't mean that it is reasonable, only that it's the safest for them.
my expectation is that the bad-is-black association is primordial enough that no amount of purging the language of related metaphors will make a dent in how people feel.

**Comment by philihp** [comment]
Long time user, first time commenter.

I would love to see something like this happen. Regardless of the racially charged etymology of blacklist and whitelist, they're still ambiguous. Allowlist and denylist are simply better descriptors, and makes a project more accessible in attracting contributions.

**Comment by JimTrout** [comment]
> my expectation is that the bad-is-black association is primordial enough that no amount of purging the language of related metaphors will make a dent in how people feel.

No disrespect to you or how you interpret this, however this is one of those situations where a simple change which really costs you nothing in the scheme of things can go a long way towards healing the hurt felt by others. It is a matter of kindness to your fellow human beings. Peace to all. Thanks.

**Comment by matthewhelmke** [comment]
https://inclusivenaming.org/ is backed by some industry giants. They have a recommendation: https://inclusivenaming.org/language/word-list/
